condensedinlinepanel/edit_handlers.py

```diff
@@ -176,7 +177,8 @@ def bind_to_model(self, model):
             'relation_name': self.relation_name,
             'related': related,
             'panels': self.panels,
-            'heading': self.label,
+            'heading': self.heading,
+            'label': self.label,
```

Comment:
I think it would be really nice to make the add label default to "add <model name>" to make it consistent with the builtin InlinePanel

Reply:
Will add that in tomorrow (hopefully).